roles/edpm_libvirt/tasks/configure.yml

```diff
@@ -64,3 +66,21 @@
     - {"src": "libvirt_virtsecretd/virtsecretd.conf", "dest": "libvirt_virtsecretd/virtsecretd.conf"}
   notify:
     - Restart libvirt
+
+
+- name: Copy TLS files to the compute node
+  tags:
+    - configure
+    - libvirt
+  become: true
+  loop:
+    - {"src": "{{ edpm_libvirt_certs_src }}/{{ inventory_hostname }}-tls.crt", "dest": "{{ edpm_libvirt_certs_dest }}/tls.crt"}
+    - {"src": "{{ edpm_libvirt_certs_src }}/{{ inventory_hostname }}-tls.key", "dest": "{{ edpm_libvirt_certs_dest }}/tls.key"}
+    - {"src": "{{ edpm_libvirt_cacerts_src }}/TLSCABundleFile", "dest": "{{ edpm_libvirt_cacerts_dest }}/TLSCABundleFile"}
+  ansible.builtin.copy:
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+    mode: '0600'
+    owner: "{{ item.owner | default(ansible_user) }}"
```

Comment:
would it be safer to force root ownership?

Reply:
in a containerized install these should be owned by root:libvirt
but in a decontaienrsied install then yes root:root

i belive root:root would work in the containerised install too so that is probably a better default in general for these files.

Reply:
we have now decontaienrised libvirt so ya root:root would make more sense.

Reply:
ack will fix